frontend/pages/_app.tsx

```diff
@@ -74,7 +74,7 @@ const App = ({ Component, pageProps }: AppPropsWithLayout) => {
       <ThemeProvider attribute="class">
         <DefaultSeo
           dangerouslySetAllPagesToNoIndex={!IS_PRODUCTION}
-          titleTemplate="%s | Flathub"
+          titleTemplate="%s | {t('Flathub')}"
```

Comment:
```suggestion
          titleTemplate={`%s | ${t("Flathub")}`}
```

The old code doesn't work, it should be fine like this